app-frontend/src/app/components/publishModal/publishModal.controller.js

```diff
@@ -1,9 +1,15 @@
 export default class PublishModalController {
-    constructor(projectService, $log) {
+    constructor(projectService, $log, tokenService, authService, $uibModal) {
         'ngInject';
 
+        this.authService = authService;
         this.projectService = projectService;
         this.$log = $log;
+        this.tokenService = tokenService;
+        this.$uibModal = $uibModal;
+
+        // TODO change this when we implement orgs
```

Comment:
I'll leave this until we update users to use a UUID and the full /users/me endpoint. Right now we don't have a great way to get the users's org on the frontend here